test/helpers/fixture_deploy_helper.rb

```diff
@@ -51,12 +51,14 @@ def deploy_dir(dir, wait: true, allow_protected_ns: false, prune: true, bindings
       current_sha: SecureRandom.hex(6),
       context: KubeclientHelper::MINIKUBE_CONTEXT,
       template_dir: dir,
-      wait_for_completion: wait,
-      allow_protected_ns: allow_protected_ns,
-      prune: prune,
+      logger: test_logger,
       bindings: bindings
     )
-    runner.run
+    runner.run(
+      skip_wait: !wait,
```

Comment:
I preferred `wait_for_completion` instead of `skip_wait` here because I usually try to avoid double negation in flag names (`disabled = true` vs `enabled = false`). What do you think?